src/app/metadata/features/dataset-expansion-panel/dataset-expansion-panel.component.html

```diff
@@ -1,95 +1,210 @@
-          track platform.value
-        ) {
-          <li class="uppercase">{{ platform.value }} {{ platform.count }}</li>
-        }
-      </ul>
-    }
+      <p><strong>Full title:</strong> {{ summary().title }}</p>
+      <p><strong>Description:</strong> {{ summary().description }}</p>
+      @if (summary().type && summary().type !== []) {
+        <p>
+          <strong>Types: </strong>
+          @for (type of summary().type; track type) {
+            <mat-chip class="mr-1 text-sm">{{ type }}</mat-chip>
+          }
+        </p>
+      }
+      <div class="mb-4 mt-8 flex pe-[15%]">
+        <div class="flex grow flex-col gap-y-8">
+          <div class="flex">
+            <div class="me-3 pt-4 text-5xl text-neutral-700">
+              <mat-icon inline>book_2</mat-icon>
+            </div>
+            <div>
+              <p>
+                <strong>Part of study: </strong>
+              </p>
+              <p>
+                {{
+                  summary().studies_summary.count > 0
+                    ? summary().studies_summary.stats.title
+                    : 'None'
+                }}
+              </p>
+              <p>
+                <strong>Accession ID: </strong>
+              </p>
+              <p>
+                {{
+                  summary().studies_summary.count > 0
+                    ? summary().studies_summary.stats.accession
+                    : 'N/A'
+                }}
+              </p>
+            </div>
+          </div>
+          <div class="flex">
+            <div class="me-3 pt-4 text-5xl text-neutral-700">
+              <mat-icon inline>genetics</mat-icon>
+            </div>
+            <div>
+              <p><strong>Sample info:</strong></p>
+              <p>
+                <strong>{{ summary().samples_summary.count }}</strong> Sample{{
+                  summary().samples_summary.count === 1 ? '' : 's'
+                }}
+                @if (summary().samples_summary.stats.sex !== []) {
+                  (Sex:
+                  @for (
+                    sex of summary().samples_summary.stats.sex;
+                    track sex.value;
+                    let last = $last
+                  ) {
+                    {{ sex.count }} {{ sex.value }}{{ last ? ')' : ', ' }}
+                  }
+                }
+              </p>
+              <p>
+                <strong>{{ summary().samples_summary.stats.tissues.length }}</strong>
+                Tissue{{
+                  summary().samples_summary.stats.tissues.length === 1 ? '' : 's'
+                }}
+              </p>
+              @if (summary().samples_summary.stats.tissues.length > 0) {
+                <ul class="ms-6">
+                  @for (
+                    tissue of summary().samples_summary.stats.tissues;
+                    track tissue.value
+                  ) {
+                    <li>
+                      <mat-chip class="mb-0.5 text-xs"
```

Comment:
It's a bit unclear to me why we sometimes use chips and sometimes ordinary bullet points. But it is like that in the legacy app and this is our reference for now. We may need to revisit this later.

Reply:
If the four sections would be more uniform (e.g. not one using chips and one bullet points), we could encapsulate them in components to avoid repetition. In the future we should do that more.

Reply:
Anyway, no action needed right now.